### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by dawnerd**

### CODE REVIEW

**Comment by dawnerd** [comment]
Not sure why travis is enabled when there's no travisfile

File: conf/default-routes.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+plugins:
+  hapi-views:
+    dataPath: '{{CWD}}'
+    enableCache: '{{ENV.ENABLE_CACHE}}'
```
Comment (jgallen23) [inline conf/default-routes.yaml]: I don't think we need cache for this

File: package.json [hunk 2]
```diff
@@ -21,25 +21,31 @@
   },
   "homepage": "https://github.com/firstandthird/healthcheck#readme",
   "dependencies": {
-    "async": "2.1.5",
+    "async": "2.4.0",
     "flat-file-db": "1.0.0",
+    "hapi-auto-handler": "1.4.1",
     "hapi-method-scheduler": "firstandthird/hapi-method-scheduler#d5852f2248a2cbf199c119356b6e22bed1974769",
-    "js-yaml": "3.8.1",
+    "hapi-nunjucks-helpers": "3.1.1",
```
Comment (jgallen23) [inline package.json]: are these used?

File: package.json [hunk 3]
```diff
@@ -21,25 +21,31 @@
...
     "lodash.minby": "4.6.0",
     "lodash.sumby": "4.6.0",
     "logr-slack": "1.1.0",
-    "moment": "2.17.1",
+    "moment": "2.18.1",
     "ping": "0.2.2",
-    "rapptor": "1.10.0",
-    "wreck": "10.0.0",
-    "yargs": "6.6.0"
+    "rapptor": "1.13.0",
```
Comment (jgallen23) [inline package.json]: I think you need to change some settings to work with logr flat for this

File: conf/default.yaml [hunk 4]
```diff
@@ -1,8 +1,15 @@
 plugins:
   hapi-method-loader:
     _priority: -6
+  vision:
+    _priority: -5
   './lib/healthcheck':
     _priority: -5
+  inert:
```
Comment (jgallen23) [inline conf/default.yaml:8]: do we need inert? doesn't seem like you are doing any static file serving

File: package.json [hunk 5]
```diff
@@ -21,25 +21,31 @@
   },
   "homepage": "https://github.com/firstandthird/healthcheck#readme",
   "dependencies": {
-    "async": "2.1.5",
+    "async": "2.4.0",
     "flat-file-db": "1.0.0",
+    "hapi-auto-handler": "1.4.1",
```
Comment (jgallen23) [inline package.json]: I don't see auto handler being used anywhere

**Review by jgallen23 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**